mongo/options/autoencryptionoptions.go

```diff
@@ -20,9 +20,6 @@ package options
 //
 // Enabling Client Side Encryption reduces the maximum document and message size (using a maxBsonObjectSize of 2MiB and
 // maxMessageSizeBytes of 6MB) and may have a negative performance impact.
-//
```

Comment:
This is the only spot left marked as beta?

Reply:
Talked offline. There were three spots where the beta marking was removed in this PR.